### PR TITLE
Upgrade docusaurus to 2.0.0.beta.14 from beta.9

### PR DESCRIPTION
**Pull request by chriswk**

### CODE REVIEW

**Comment by ivarconr** [comment]
Why is the build failing?

**Comment by thomasheartman** [comment]
I think @chriswk said it was because the `/integrations/integrations` page now lives at `/integrations` instead. 

The reason for this seems to be [this PR in the docusaurus repo](https://github.com/facebook/docusaurus/pull/5830). From what I gather, there is a change that'll make pages called `index`, `readme` and the name of the subdirectory (e.g. `/integrations/integrations`) automatically get rendered at the subdir path (i.e. `/integrations`). 

The reason for the above change is that categories can now have actual docs instead of being just containers. This means that instead of having an 'introduction' or 'overview' doc as the first subdoc of each category, the overview will now show when you click the category itself.

The link rewriting is actually in line with how we've done most of our docs (with index docs listed at the directory url). I also like the idea of not needing the 'overview' docs, but would be happy to discuss that. 

Anyway, with that in mind, I'll see if I can't fix the build issues. Docusaurus recommends you link to the doc file instead of to a URL to get around issues like this, so I'll make some updates 🙋🏼

**Review by thomasheartman — APPROVED** [review]
Looks good to me 🙋🏼

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -146,6 +146,10 @@ module.exports = {
                         to: '/advanced/toggle_variants',
                         from: '/toggle_variants',
                     },
+                    {
+                        to: '/integrations'
+                        from: '/integrations/integrations'
```
[the remark this answers is not in the document]
Comment (thomasheartman) [reply website/docusaurus.config.js]: @sonatype-lift ignore

You seem to have been confused with previous versions of the diff.

**Pull request merged**